server/server.go

```diff
@@ -2571,7 +2571,7 @@ func (c *tlsMixConn) Read(b []byte) (int, error) {
 	return c.Conn.Read(b)
 }
 
-func (s *Server) createClient(conn net.Conn) *client {
+func (s *Server) createClient(conn net.Conn, inProcess bool) *client {
```

Comment:
Prefer a new func called createInProcessClient(), keep the create client the way it is and add in another that takes the bool flag..

Reply:
We will be formalizing alot on clients, so might be worth it imo.

Reply:
Sure, have done.